llarp/router/router.cpp

```diff
@@ -697,7 +697,10 @@ namespace llarp
       connected += _linkManager.NumberOfPendingConnections();
     }
 
-    _rcLookupHandler.ExploreNetwork();
+    const int interval = isSvcNode ? 30 : 5;
```

Comment:
Instead of this we can keep the unconditional ExploreNetwork() here, and then just skip ticks inside ExploreNetwork to throttle lookups (but still distribute them over time).  (See my comments above).